W-14073894 close project duke (#154)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-open-packaged-sources.adoc

//tags are for upload-a-project.adoc

//
// tag::close-folder[]
// step before importing

> . If a folder is open in the Explorer, close the folder by selecting *File* > *Close Folder* from the IDE.

Reviewer: Just flagging to make sure the numbering is correct here, since the next one starts with 1.

Author: the sequencing works, thx for flagging though

+
The Explorer provides an *Open Folder* button if no folder is open.
+

> This step prevents the import command from adding the project as a subfolder of an open folder and enables the import process to open your project folder automatically.

Author: ```suggestion
This step prevents the import command from adding the project as a subfolder of an open folder and enables the import process to open your imported project folder automatically in the IDE.
```

> This step prevents the import command from adding the project as a subfolder of an open folder and enables the command to open your project folder automatically after importing.

Author: ```suggestion
This step prevents the import command from adding the project as a subfolder of an open folder and enables the import process to open your project folder automatically.
```

> This step prevents the import command from adding the project as a subfolder of an open folder and enables the import process to open your imported project folder automatically in the IDE.

Author: ```suggestion
This step prevents the import process from adding the project as a subfolder of an open folder and enables the import process to open your imported project folder automatically in the IDE.
```
